.expeditor/config.yml

```diff
@@ -0,0 +1,15 @@
+# Documentation available at https://expeditor-docs.es.chef.io/
+
+github:
+  # The file where the MAJOR.MINOR.PATCH version is kept. The version in this file
+  # is bumped automatically via the `built_in:bump_version` merge_action.
+  version_file: "VERSION"
```

Comment:
This is fine for now, but Expeditor currently does not support the `-dev` suffix. If this is something that you strongly want/need, let me know.